src/app/models/propTypes/general/index.js

```diff
@@ -0,0 +1,12 @@
+import { shape, string, oneOf, arrayOf, oneOfType } from 'prop-types';
+
+export const blockOfSpecificTypeAndModel = (typeEnum, model) =>
```

Comment:
I'm not sure about the naming of `typeEnum`, this suggests we have a definition of constants which state all the different blocks we can have. Unless we plan to create this enum I personally think this param could be `blockName`?

Reply:
There may be benefit in the enum but we don't have it now. Will change.